Packs/CommonScripts/Scripts/CheckPDFEncryptionAndValidity/CheckPDFEncryptionAndValidity.yml

```
@@ -0,0 +1,36 @@
+args:
+- default: false
+  description: The entry ID of the PDF file.
+  isArray: false
+  name: EntryID
```

```suggestion
- name: EntryID
  default: false
  description: The entry ID of the PDF file.
  isArray: false

```

Changing the order of fields will allow us minimize irrelevant lines easily.